openshift-install-powervs

```diff
@@ -656,6 +658,22 @@ function precheck_tools {
   fi
 }
 
+#-------------------------------------------------------------------------
+# Check Flavor and run setup
+#-------------------------------------------------------------------------
+function flavor_set {
+
+#  Flavor Option
+  if [ -f "logs/flavor.log" ]; then
```

Comment:
Why new log file?

Reply:
It checks if the flavor is specified; if not it asks for the  flavor related interactive questions